src/packages/cli/src/index.ts

```diff
@@ -407,7 +407,7 @@ yargs
 	.command({
 		command: '*',
 		handler() {
-			yargs.showHelp();
+			yargs().showHelp();
```

Comment:
Call the help on your yargs instance, not create another one.

```suggestion
			yargsInstance.showHelp();
```